schema/bom-1.6.proto

```diff
@@ -148,6 +148,10 @@ message Component {
   repeated OrganizationalContact authors = 29;
   // Textual strings that aid in discovery, search, and retrieval of the associated object. Tags often serve as a way to group or categorize similar or related objects by various attributes. Examples include "json-parser", "object-persistence", "text-to-image", "translation", and "object-detection".
   repeated string tags = 30;
+  // Specifies the OmniBOR Artifact ID. The OmniBOR, if specified, MUST be valid and conform to the specification defined at: https://www.iana.org/assignments/uri-schemes/prov/gitoid
+  optional string omnibor = 31;
```

Comment:
If I understand correctly the same package url identified by a version or tag could have several gitoid which is based on git commit hashes.

Reply:
Can you reach out to the OmniBOR folks and confirm. If we need to make this an array, its an easy change.

Reply:
While a purl could have multiple OmniBORs, how about the specific component used? If I have a single component, with or without a purl, can a component have multiple gitoids? I didn't think this was possible as I thought a single artifact could only have a single gitoit.

Reply:
https://omnibor.io/glossary/omnibor/#omnibor-identifier

As per this doc, every class, .so or, source code would have its own ID. It is not clear if there is a single parent artifact id that can be used without listing the child id. Best to ask the omnibor team directly for some examples and clarifications